docker/DOCKERFILE

```diff
@@ -0,0 +1,20 @@
+ADD file ... in /
+CMD ["/bin/sh"]
+ENV NODE_VERSION=12.16.1
+/bin/sh -c addgroup -g 1000
+ENV YARN_VERSION=1.22.0
+/bin/sh -c apk add --no-cache
+COPY file:238737301d47304174e4d24f4def935b29b3069c03c72ae8de97d94624382fce in /usr/local/bin/
```

Comment:
Where is this file from?

Reply:
i just copied and pasted it from https://hub.docker.com/r/patwoz/expo-cli